### DISCUSSION TITLE
New feature to show status as always online.

### DISCUSSION THREAD

**Discussion by gokunik** [tomer8007/whatsapp-web-incognito#50] [Ideas]
The extension works really well and has really good features. Can you also add this new feature which will always show the status as online. As you know that once we switch the window it will show our status as offline. I want that as long as the browser window is open it should show status as online.
Would be happy if this feature is implemented. By the time I am trying to understand the code if I can help in anyway 😁.

**Comment by tomer8007** [comment]
Why would you want such a feature?

**Comment by gokunik** [comment]
It is helpful in certain situations.
Mainly I want it because sometimes I just need such a feature where I can show myself online even if I am working on the other window. Because, usually, my whats app window remains always open side by side with the other chrome window, and sometimes it's annoying for other people when it happens. While working, it happens with a lot of people that they switch windows while they are talking to a person because talking from a computer is a lot faster when you have really good typing speed. This thing is becoming quite noticeable by other people and even they told me to stop doing it else they will not talk to me because it is annoying for them that I am going off and on again and again when all I was doing is switching to another window until they write their message. Even I also agree when someone talks like this it's annoying when it feels that someone is going offline again and again. I never go offline just change the window but people feel I am going offline which causes the problem. 

Now you would say why change the windows then? If you are talking to someone first talk and then change the window. Well, in reality, things are not that simpler. I type fast and usually, I have to wait for another person to send its the complete message and if I am working or using other things on the computer then I switch windows until the person sends his message, like checking emails, notification on Linkedin, sometimes checking Twitter or surfing youtube side by side to change the music or find some good music and such task don't require too much attention so I just keep my self jumping from one window to another. I think you will get the point. And this is what happens with many people so it can be a useful feature. this feature will only add more flexibility.

I never faced this problem before but after covid, many things have changed.

I guess I may not be the only person who faces this problem because while finding a solution to this problem, I have come across many people looking for the same thing for the same or other reasons.


I hope you add this feature is possible 😁. Your efforts will be appreciated 👏.

**Comment by tomer8007** [comment]
It's kind of the opposite of being incognito, but I think you should open an issue for this so I can keep this feature request in mind.

**Reply by gokunik** [reply]
Yeah, that it is but I think it will be a useful feature. Also, it depends on the person what they want so they can enable and disable the options accordingly.
Thank you for considering it. 😁
